pkgs/tools/misc/neofetch/default.nix

```diff
@@ -2,25 +2,28 @@
 
 stdenv.mkDerivation rec {
   name = "neofetch-${version}";
-  version = "3.0.1";
+  version = "3.2.0";
   src = fetchFromGitHub {
     owner = "dylanaraps";
     repo = "neofetch";
     rev = version;
-    sha256 = "0ccdgyn9m7vbrmjlsxdwv7cagsdg8hy8x4n1mx334pkqvl820jjn";
+    sha256 = "1skkclvkqayqsbywja2fhv18l4rn9kg2da6bkip82zrwd713akl3";
   };
 
-  patchPhase = ''
-    substituteInPlace ./neofetch \
-    --replace "/usr/share" "$out/share"
-  '';
+  # This patch is only needed so that Neofetch 3.2.0 can look for
+  # configuration file, w3m directory (for fetching images) and ASCII
+  # directory properly. It won't be needed in subsequent releases.
+  patches = [
+  ./config.patch
```

Comment:
Is this patch available over http? Generally we prefer to use `fetchpatch` over embedding the patch in the repo.